env_vars.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Environment variables for build
+MACOSX_DEPLOYMENT_TARGET=10.9
+# CFLAGS override for https://github.com/pandas-dev/pandas/issues/34114
+CFLAGS="-std=c99 -Wno-error=maybe-uninitialized -Wno-error=sign-compare"
```

Comment:
No problem with `maybe-uninitialized` if it gets things through, but I think `sign-compare` is a pretty serious issue that can easily show up for end users so not something we should ignore

Reply:
I agree, but I don't actually know how to do that. The wheel-building repo feels like the wrong place to ensure that we're writing clean C code.

Once I get a build passing here, I'll try to add a CI job on pandas-dev/pandas that at least compiles things with the manylinux1 docker image.